src/components/Tabs/tabs.module.scss

```diff
@@ -129,23 +129,94 @@
         background-color: var(--tab-pill-background);
         width: fit-content;
         border-radius: $corner-radius-xl;
-        padding: $space-xs;
+        padding: calc(#{$space-xs} - #{$space-xxxs});
```

Comment:
why did this become complex? ... since they are both known values suggest letting sass do the math instead of the browser

Reply:
the design calls for a 6px padding at this size :) no var exists for that amount.